deployments/UnityMixpanel/Assets/Mixpanel/Editor/Mixpanel/MixpanelPostprocessor.cs

```diff
@@ -1,31 +1,66 @@
-﻿#if UNITY_IPHONE
+﻿#if UNITY_IOS
 using UnityEngine;
 using UnityEditor;
 using UnityEditor.Callbacks;
+using MixpanelSDK.UnityEditor.iOS.Xcode;
 using System;
 using System.Diagnostics;
+using System.Collections;
+using System.IO;
 
 public class MixpanelPostprocessScript : MonoBehaviour
 {
     [PostProcessBuild]
-    public static void OnPostprocessBuild(BuildTarget target, string pathToBuildProject)
+    public static void OnPostprocessBuild(BuildTarget target, string buildPath)
     {
         UnityEngine.Debug.Log("******** START Mixpanel iOS Postprocess Script ********");
 
-        var scriptSymbols = PlayerSettings.GetScriptingDefineSymbolsForGroup(BuildTargetGroup.iOS);
-
-        Process process = new Process();
-        process.StartInfo.FileName = "python";
-        process.StartInfo.Arguments = string.Format("Assets/Mixpanel/Editor/Mixpanel/post_process.py \"{0}\" \"{1}\"", pathToBuildProject, scriptSymbols);
-        process.StartInfo.UseShellExecute = false;
-        process.StartInfo.RedirectStandardOutput = true;
-        process.StartInfo.CreateNoWindow = true;
-        process.Start();
-        while (!process.StandardOutput.EndOfStream) {
-            UnityEngine.Debug.Log(process.StandardOutput.ReadLine());
+        // Native Xcode project support
+        if (target == BuildTarget.iOS)
+        {
+            // Find the xcodeproj based on the build path
+            string projectPath = PBXProject.GetPBXProjectPath(buildPath);
+
+            // Load and parse the xcodeproj
+            PBXProject project = new PBXProject();
+            project.ReadFromFile(projectPath);
+
+            // Find the default (non-test) target
+            string targetGuid = project.TargetGuidByName(PBXProject.GetUnityTargetName());
+
+            // Perform our customizations to their xcodeproj
+            AddLinkerFlags(project, targetGuid);
+            AddFrameworks(project, targetGuid);
+            EnableBitcode(project, targetGuid);
+            // Remove OSX bundle to work around Unity 4.X bug that incorrectly imports the
+            // OSX bundle in iOS projects
+            RemoveMacBundle(project);
+
+            project.WriteToFile (projectPath);
         }
 
         UnityEngine.Debug.Log("******** END Mixpanel iOS Postprocess Script ********");
     }
+
+    private static void AddLinkerFlags(PBXProject project, string targetGuid)
+    {
+        project.SetBuildProperty(targetGuid, "OTHER_LD_FLAGS", "$(inherited) -lc++");
```

Comment:
shouldn't stuff like this be in the Makefile? Is this the unity way of handling it?